frontend/translations/en.po

```diff
@@ -6675,6 +6675,10 @@ msgstr "Invalid token value: %s"
 msgid "workspace.token.opacity-range"
 msgstr "Opacity must be between 0 and 100% or 0 and 1 (e.g. 50% or 0.5)."
 
+#: src/app/main/ui/workspace/tokens/errors.cljs
+msgid "workspace.token.stroke-width-range"
+msgstr "Stroke width must be between equal or bigger than 0."
```

Comment:
This is just a comment, you don't need to change it for now:
I think this should be under errors...  `errors.tokens.stroke-width-range` and many other keys.
We clearly need to refactor this.